ur_description/config/ur5e/visual_parameters.yaml

```diff
@@ -1,67 +1,67 @@
 mesh_files:
   base:
     visual:
-      mesh: "package://ur_description/meshes/ur5e/visual/base.dae"
+      mesh: "ur5e/visual/base.dae"
       material:
         name: "LightGrey"
         color: "0.7 0.7 0.7 1.0"
```

Comment:
Why are we specifying the link material here? The .dae files are already colored.

I I delete the `material` variables from this file and `ur_common.xacro` and `ur_macro.xacro` then the robot appears in simulation and Rviz as expected.
![image](https://user-images.githubusercontent.com/25058794/144717353-2791ee13-4fce-4c4a-8ba2-4920b21c1e36.png)

Reply:
This is a porting “relict” from ROS 1. We should check this at least for the visuals. I would open a follow-up issue for this since it is not really simulation related. @livanov93 @urrsk what do you think? Would should be done here? Should we ignore materials for visuals in the ur_macro driver?